manifest.toml

```
@@ -0,0 +1,57 @@
+packaging_format = 2
+
+id = "element"
+name = "Element"
+description.en = "Web client for Matrix"
+description.fr = "Client web pour Matrix"
+
+version = "1.11.23~ynh1"
+
+maintainers = ["Josué Tille"]
+
+[upstream]
+license = "Apache-2.0"
+website = "https://element.io"
+demo = "https://app.element.io/"
+admindoc = "https://element.io/help"
+code = "https://github.com/vector-im/element-web/"
+cpe = "cpe:2.3:a:matrix:element"
+
+[integration]
+yunohost = ">= 11.1.6"
+architectures = "all"
+multi_instance = true
+ldap = false
+sso = true
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/element"
+
+    [install.default_home_server]
+    ask.en = "Choose a default home server"
+    ask.fr = "Choisissez un serveur par défault"
+    type = "string"
+    example = "my-own-homeserver.tld"
+    default = "matrix.org"
+
+    [install.init_main_permission]
+    help.en = "A public application means that anyone can access it. Note that this application provides just files (html, javascript, images, etc.)"
+    help.fr = "Une application publique signifie que n'importe qui peut y accéder. Notez que cette application ne fournit que des fichiers (html, javascript, images, etc.)"
```

```suggestion
    help.en = "A public application means that anyone can access it. Note that this application is only a web frontend that connects to a Synapse account on other servers"
    help.fr = "Une application publique signifie que n'importe qui peut y accéder. Notez que cette application n'est qu'une interface web qui se connecte à un compte Synapse sur d'autres serveurs"
```

Not 100% sure how to formulate this but the original message is kinda confusing